examples/pytorch/ogb/deepwalk/deepwalk.py

```diff
@@ -247,26 +247,29 @@ def fast_train(self):
                 if i > 0 and i % self.args.print_interval == 0:
                     if self.args.print_loss:
                         print(
-                            "Batch %d training time: %.2fs loss: %.4f"
+                            "\rBatch %d training time: %.2fs loss: %.4f"
```

Comment:
why this change?

Reply:
revert?

Reply:
Without this change, we do have more than 1100 lines (1138, to be exact) similar to the following one:
```
Batch 93700, training time: 1.49s
```
I think it would be better to just output this information to the same line of the screen and change this line about every 1.5 seconds.

Reply:
Usually for example in DGL, we output newline without erase the previous. And we use i % self.args.print_interval = 0 to reduce the frequency if needed.

Reply:
Reverted